### PR TITLE
added document on securing AMP docs with CSP

### PR DESCRIPTION
**Pull request by CrystalOnScript**
Resolves [issue #1316](https://github.com/ampproject/amphtml/issues/1316)

### CODE REVIEW

**Comment by sebastianbenz** [comment]
We should add this to amp.dev as well.

**Comment by CrystalOnScript** [comment]
@sebastianbenz this PR is adding a document to amp.dev, do you meaning adding [this file](https://github.com/ampproject/amphtml/blob/master/examples/csp.amp.html) as a demo?

**Comment by sebastianbenz** [comment]
Sorry, I meant amp.dev itself should be served with these CSP headers...

**Comment by CrystalOnScript** [comment]
HA!

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/secure-pages.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+$title: Secure from third party attacks 
+$order: 5
+description: Take measures to protect your AMP pages and users from security vulnerabilities on the web
+  - websites
+author: CrystalOnScript
+---
+
+Take measures to protect your site and users from security vulnerabilities on the web. One of the most sinister is [cross-site scripting](https://www.google.com/about/appsecurity/learning/xss/) (XSS). XSS is a security bug that can allow an attacker to inject malicious code onto the HTML pages displayed to users. 
+
+Protect against these types of attacks by adopting a [Content Security Policy (CSP)](https://csp.withgoogle.com/docs/index.html). AMP already adds a CSP to cached AMP pages! However, pages lack this additional layer of protection when users circumvent the cached version, if you do not add your own CSP.
```
Comment (pbakaus) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/secure-pages.md]: "AMP caches like the Google AMP Cache already add CSP to your pages!"

(it's not AMP that adds it)

**Pull request merged**